hijack/templates/hijack/notification.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n static %}
+
+<link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack.min.css' %}" media="screen">
```

Comment:
`hijack.min.css`
![image](https://user-images.githubusercontent.com/721963/110682090-9b6f6a80-81eb-11eb-997e-f04c884f68c6.png)

Reply:
You'll need to run `make dist` to build the static assets. However, I also added a GitHub action, that adds the entire dist build as an artifact. Makes it easy to download and install.

Reply:
The problem is that you can't just install the package directly from github that way

Reply:
https://github.com/django-hijack/django-hijack/blob/e782910ade1753a100c3711c6ced6b437ab7dd9e/package.json#L5-L6
@codingjoe The file name is different from build file

Reply:
Yes, sorry, that was a rebasing error. I corrected the `package.json`.